tools/crm_resource.c

```diff
@@ -1873,8 +1873,8 @@ main(int argc, char **argv)
             break;
 
         case cmd_locate: {
-            GListPtr resources = cli_resource_search(out, rsc, options.rsc_id, data_set);
-            rc = out->message(out, "resource-search-list", resources, rsc, options.rsc_id);
+            GListPtr nodes = cli_resource_search(out, rsc, options.rsc_id, data_set);
```

Comment:
Just for clarity, what about changing the name here too?
https://github.com/ClusterLabs/pacemaker/blob/master/tools/crm_resource_runtime.c#L1783

Reply:
Done.